test/integration/viz/policy/testdata/emoji-policy-authorize-all.yaml

```
@@ -0,0 +1,75 @@
+---
+apiVersion: policy.linkerd.io/v1beta1
+kind: Server
+metadata:
+  name: emoji-grpc
+  labels:
+    app.kubernetes.io/part-of: emojivoto
+    app.kubernetes.io/name: emoji
+    app.kubernetes.io/version: v11
+spec:
+  podSelector:
+    matchLabels:
+      app: emoji-svc
+  port: grpc
+  proxyProtocol: gRPC
+---
+apiVersion: policy.linkerd.io/v1beta1
```

Should this be `v1alpha1`? I think after re-introducing AuthorizationPolicies we went back to alpha1.

```suggestion
apiVersion: policy.linkerd.io/v1alpha1
```

Think it might be what triggered the test failure. I think on `Server` resources we're still on `v1beta1` though.